src/main/java/com/matchbook/sdk/StreamObserver.java

```diff
@@ -0,0 +1,11 @@
+package com.matchbook.sdk;
+
+import com.matchbook.sdk.exceptions.MatchbookSDKException;
+
+public interface StreamObserver<V> {
```

Comment:
What ```V``` stands for? :slightly_smiling_face:

Reply:
Well, nothing special :)